lib/sqlalchemy/orm/session.py

```diff
@@ -152,7 +152,7 @@
 _BindArguments = Dict[str, Any]
 
 _EntityBindKey = Union[Type[_O], "Mapper[_O]"]
-_SessionBindKey = Union[Type[Any], "Mapper[Any]", "Table"]
+_SessionBindKey = Union[Type[Any], "Mapper[Any]", "Table", str]
```

Comment:
I'm going to change this however it requires that I change the code to not use ``__binds`` for this purpose, so I'll do that here.